strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/ConfigurationManagementServiceImpl.java

```diff
@@ -618,4 +624,38 @@ private void modifyInLock(final Consumer<MutableConfiguration> operation,
         }
     }
 
+    @Override
+    public boolean setMaxUploadFileSize(String maxFileSizeString) throws SizeFormatValidationException
+    {
+
+        if(this.multipartConfigElement instanceof UpdatableMultipartConfigElement){
+            Long parsedSize = parseUploadSize(maxFileSizeString);
+            ((UpdatableMultipartConfigElement) multipartConfigElement).setMaxFileSize(parsedSize);
+            return true;
+        }
+
+        return false;
+    }
+
+    private Long parseUploadSize(String maxFileSizeString) throws SizeFormatValidationException
+    {
+
+        maxFileSizeString = maxFileSizeString.toUpperCase();
+        String sizeUnit = maxFileSizeString.replaceAll("[^A-Z]", "");
+        Long size = Long.parseLong(maxFileSizeString.replaceAll("[^0-9]", ""));
+
+        int pow;
+        switch (sizeUnit){
+            case "B": {pow = 0; break;}
+            case "KB": {pow = 1; break;}
+            case "MB": {pow = 2; break;}
+            case "GB": {pow = 3; break;}
+            default: throw new SizeFormatValidationException(sizeUnit + " is not acceptable upload size unit");
+        }
```

Comment:
Cosmetic: Coding convention:
```suggestion
        switch (sizeUnit){
            case "B":
                pow = 0; break;
            case "KB": 
                pow = 1; break;
            case "MB":
                pow = 2; break;
            case "GB":
                pow = 3; break;
            default:
                throw new SizeFormatValidationException(sizeUnit + " is not acceptable upload size unit");
        }
```